DolbyIO/DolbyIO.Android.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<root xmlns:android="http://schemas.android.com/apk/res/android">
+  <init>
+    <log text="DolbyIO plugin init"/>
+  </init>
+
+  <gradleProperties>
+    <insert>
+org.gradle.jvmargs=-Xmx1536M --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
+ANDROID_BUILD_TOOLS_GRADLE_VERSION=com.android.tools.build:gradle:7.3.1
+ANDROID_TOOLS_BUILD_GRADLE_VERSION=com.android.tools.build:gradle:7.3.1
+    </insert>
+  </gradleProperties>
+
+  <soLoadLibrary>
+    <loadLibrary name="dolbyio_comms_sdk_android_cppsdk" 
+      failmsg="XML! libdolbyio_comms_sdk_android_cppsdk.so library not loaded and required!" />
+  </soLoadLibrary>
```

Comment:
Is this necessary?

Reply:
yup !

what happens is that "normally" the cppsdk is the shared object that is provided for each platforms. However Android provides some layer of configurations that are normally transparent when using the maven dependency:

- media's so
- cppsdk's so
- and the upper layer of the "loader"

but since the cppsdk's so is not depending onto the upper layer (and that's the over way around - for clarity as regular android project), we need to make sure that this platform specific layer is loaded alongside the project's configuration - and this from the documentation should be done here

(but I don't exclude that later versions of the cppsdk will hide a bit more any shenanigans related to the platform's env)